docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,435 @@
+= A simple date-clock application: tutorial cloud-harness
+
+In this small tutorial, we will see different aspects about the developement of applications with cloud harness through the developement from scratch of a small webapp that fetch information from a server on a regular basis.
+
+Cloud harness allows you to quickly setup app that needs to be ran in a kubernete cluster.
+It generates the initial files and folders for your project depending on some templates tacking different aspects of your app depending on your requirements, __e.g.__, for a webapp project, it generates the frontend initial files for ReactJS and the initial Flask files for the backend.
+For the API part, cloud harness relies on OpenAPI 3 to deal with the endpoints/model description.
+
+The different aspects that will be covered here are:
+
+* how to install cloud-harness;
+* how to quickly setup a first version of a local cluster using minikube;
+* how to build and locally deploy the projects from the repository;
+* how to bootstrap a new app, build it and run it;
+* how to modify/update the app, built it and run it again.
+
+== The tools you need
+
+Before installing everything, please be sure you have the following tools installed on your machine:
+
+* `python`
+* `minikube` -- the local mini cluster
+* `kubectl` -- to help controlling the cluster
+* `helm` -- to deal with Kubernete "packages" (helm is basically a package manager for Kubernete)
+* `skaffold` -- to build/deploy/run the apps
+
+
+== Install cloud-harness (if it is not yet done)
+
+The installation is pretty straight forward.
```

Comment:
That's just a matter of style, just to think about it: it happens I feel stupid when a tutorial says it's very simple and something does not work :sweat_smile:

Reply:
Noted, I'll be more careful, it's true it's annoying when nothing works when it's supposed to be simple :sweat_smile: